pg_upgrade_internal.log

```diff
@@ -0,0 +1,7 @@
+
+-----------------------------------------------------------------
+  pg_upgrade run on Wed Oct 13 15:44:00 2021
```

Comment:
Let's get rid of these pg_log files. Especially when we do not use psql as our database engine